### PR TITLE
refactor(project): removing packages/tokens

### PR DESCRIPTION
**Pull request by jw-foss**
Please make sure these boxes are checked before submitting your PR, thank you!

* Removing all files under `packages/tokens`
* Moving all tokens under its component directory where they should be in the first place.

Reviewers should review the changes commit by commit which simplifies the complexity of this PR.

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/popper/src/content.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { buildProps, definePropType } from '@element-plus/utils'
 
 import type { ExtractPropTypes, StyleValue } from 'vue'
 import type { Options, Placement } from '@popperjs/core'
-import type { Measurable } from '@element-plus/tokens'
+import type { Measurable } from '@element-plus/components/popper'
```
Comment (tolking) [inline packages/components/popper/src/content.ts:6]:
```suggestion
import type { Measurable } from './constants'
```

File: packages/components/popper/src/trigger.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { buildProps, definePropType } from '@element-plus/utils'
 
-import type { Measurable } from '@element-plus/tokens'
+import type { Measurable } from '@element-plus/components/popper'
```
Comment (tolking) [inline packages/components/popper/src/trigger.ts:3]:
```suggestion
import type { Measurable } from './constants'
```

File: packages/components/popper/src/utils.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { isClient, unrefElement } from '@vueuse/core'
 import type { ComponentPublicInstance } from 'vue'
 import type { MaybeRef } from '@vueuse/core'
 import type { Modifier } from '@popperjs/core'
-import type { Measurable } from '@element-plus/tokens'
+import type { Measurable } from '@element-plus/components/popper'
```
Comment (tolking) [inline packages/components/popper/src/utils.ts:6]:
```suggestion
import type { Measurable } from './constants'
```

File: packages/components/tabs/src/tab-bar.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { buildProps, definePropType, mutable } from '@element-plus/utils'
 import type { ExtractPropTypes } from 'vue'
-import type { TabsPaneContext } from '@element-plus/tokens'
+import type { TabsPaneContext } from '@element-plus/components/tabs'
```
Comment (tolking) [inline packages/components/tabs/src/tab-bar.ts:3]:
```suggestion
import type { TabsPaneContext } from './constants'
```

**Review by tolking — COMMENTED** [review]
I found that some code formats are not uniform.

File: packages/components/popper/__tests__/arrow.test.tsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { nextTick, ref } from 'vue'
 import { mount } from '@vue/test-utils'
 import { afterEach, beforeEach, describe, expect, it } from 'vitest'
-import { POPPER_CONTENT_INJECTION_KEY } from '@element-plus/tokens'
+import { POPPER_CONTENT_INJECTION_KEY } from '@element-plus/components/popper'
```
Comment (chenxch) [inline packages/components/popper/__tests__/arrow.test.tsx:4]:
```
import { POPPER_CONTENT_INJECTION_KEY } from '../src/constants'
```

File: packages/components/popper/__tests__/content.test.tsx [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import { computed, defineComponent, nextTick, ref } from 'vue'
 import { mount } from '@vue/test-utils'
 import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
-import { POPPER_INJECTION_KEY } from '@element-plus/tokens'
+import { POPPER_INJECTION_KEY } from '@element-plus/components/popper'
```
Comment (chenxch) [inline packages/components/popper/__tests__/content.test.tsx:4]: as above

File: packages/components/popper/__tests__/popper.test.tsx [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import { defineComponent, inject, nextTick } from 'vue'
 import { mount } from '@vue/test-utils'
 import { describe, expect, it } from 'vitest'
-import { POPPER_INJECTION_KEY } from '@element-plus/tokens'
+import { POPPER_INJECTION_KEY } from '@element-plus/components/popper'
```
Comment (chenxch) [inline packages/components/popper/__tests__/popper.test.tsx:4]: as above

File: packages/components/popper/__tests__/trigger.test.tsx [hunk 8]
```diff
@@ -1,7 +1,7 @@
 import { nextTick, ref } from 'vue'
 import { mount } from '@vue/test-utils'
 import { afterEach, describe, expect, it, vi } from 'vitest'
-import { POPPER_INJECTION_KEY } from '@element-plus/tokens'
+import { POPPER_INJECTION_KEY } from '@element-plus/components/popper'
```
Comment (chenxch) [inline packages/components/popper/__tests__/trigger.test.tsx:4]: as above

File: packages/components/tooltip/__tests__/content.test.tsx [hunk 9]
```diff
@@ -10,7 +10,7 @@ import {
   it,
 } from 'vitest'
 import { usePopperContainer } from '@element-plus/hooks'
-import { TOOLTIP_INJECTION_KEY } from '@element-plus/tokens'
+import { TOOLTIP_INJECTION_KEY } from '@element-plus/components/tooltip'
```
Comment (chenxch) [inline packages/components/tooltip/__tests__/content.test.tsx:13]:
```
import { TOOLTIP_INJECTION_KEY } from '../src/constants'
```
I won't mark the others.

File: packages/components/form/__tests__/form.test.tsx [hunk 10]
```diff
@@ -22,7 +22,7 @@ import FormItem from '../src/form-item.vue'
 import DynamicDomainForm, { formatDomainError } from '../mocks/mock-data'
 
 import type { VueWrapper } from '@vue/test-utils'
-import type { FormRules } from '@element-plus/tokens'
+import type { FormRules } from '@element-plus/components/form'
```
Comment (chenxch) [inline packages/components/form/__tests__/form.test.tsx:25]:
```
import type { FormRules } from '../src/constants'
```

File: packages/components/upload/__tests__/upload-dragger.test.tsx [hunk 11]
```diff
@@ -1,7 +1,7 @@
 import { computed, provide } from 'vue'
 import { mount } from '@vue/test-utils'
 import { describe, expect, test, vi } from 'vitest'
-import { uploadContextKey } from '@element-plus/tokens'
+import { uploadContextKey } from '@element-plus/components/upload'
```
[the remark this answers is not in the document]
Comment (jw-foss) [reply packages/components/upload/__tests__/upload-dragger.test.tsx:4]: @chenxch this was actually fixed in [f427e1d](https://github.com/element-plus/element-plus/pull/11769/commits/f427e1dcfd47f6bb1c488791b31ecd47966191ce)

File: packages/components/popper/__tests__/arrow.test.tsx (hunk 5 above)
Comment (jw-foss) [reply packages/components/popper/__tests__/arrow.test.tsx:4]: Same.

File: packages/components/form/__tests__/form.test.tsx (hunk 10 above)
Comment (jw-foss) [reply packages/components/form/__tests__/form.test.tsx:25]: I used absolute path in tests for all components,I think this is just fine.

File: packages/components/popper/src/content.ts (hunk 1 above)
Comment (jw-foss) [reply packages/components/popper/src/content.ts]: Done, and for rest of this thread.

**Review by tolking — APPROVED** [review]

**Review by chenxch — APPROVED** [review]

**Pull request merged**